query/security/ldap/src/main/java/org/codice/ddf/admin/ldap/persist/CreateLdapConfiguration.java

```diff
@@ -146,6 +146,7 @@ public BooleanField performFunction() {
 
   @Override
   public void validate() {
+    config.connectionField().isRequired(true);
     String useCase = config.settingsField().useCase();
```

Comment:
❓ Thoughts on adding validation here for the `loadBalancing` field and the `connections` field? I assume `loadBalancing` must be specified when more than one connection is provided. It also sounds incorrect to send a `loadBalancing` field when there is only a single connect.